src/docfx/lib/markdown/MarkdownEngine.cs

```diff
@@ -321,12 +321,12 @@ private string GetImageLink(string path, MarkdownObject origin, string? altText)
                 altText = ToPlainText(origin);
             }
 
-            return GetImageLink(new SourceInfo<string>(path, origin.GetSourceInfo()), altText);
+            return GetImageLink(new SourceInfo<string>(path, origin.GetSourceInfo()), origin, altText);
         }
 
-        private string GetImageLink(SourceInfo<string> href, string? altText)
+        private string GetImageLink(SourceInfo<string> href, MarkdownObject? origin, string? altText)
```

Comment:
Should be able to remove all the `?` touched in this PR.

```suggestion
        private string GetImageLink(SourceInfo<string> href, MarkdownObject origin, string? altText)
```

Reply:
OK. Got it. Thanks a lot.